.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
add `.DS_Store` to the `.gitignore` and force push to remove it.

Reply:
Thats the gitignore file thats changed